### ISSUE TITLE
Code sample didn't work until I modified it

### ISSUE THREAD

**Issue by alannwilliams** [MicrosoftDocs/azure-docs-cli#4037]
### Prerequisites

- [X] Write a descriptive title.
- [X] Search the existing issues.
- [X] I am reporting the documentation problem for the current version of Azure CLI.

### Azure CLI version being used

2.55.0

### Link to affected document

https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=powershell

### Description of the documentation error

The powershell script to install Azure CLI did not work on my machine:
$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows64 -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
No error was returned and I couldn't run az commands
I removed the silently continue and the /quiet flag and it installed perfectly (albeit with popups and required click thrus)
Invoke-WebRequest -Uri https://aka.ms/installazurecliwindowsx64 -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi'; Remove-Item .\AzureCLI.msi

### Suggested fix

Invoke-WebRequest -Uri https://aka.ms/installazurecliwindowsx64 -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi'; Remove-Item .\AzureCLI.msi

**Comment by RamanathanChinnappan-MSFT** [comment]
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8246b90a-efb1-03ce-f642-d3eb2441fc30
* Version Independent ID: 2fdd20ef-7bd3-57bc-7957-93f4bb5377ce
* Content: [Install the Azure CLI for Windows](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=powershell)
* Content Source: [docs-ref-conceptual/install-azure-cli-windows.md](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/install-azure-cli-windows.md)
* Service: **azure-cli**
* GitHub Login: @jiasli
* Microsoft Alias: **jiasli**

**Comment by dbradish-microsoft** [comment]
Hello, @alannwilliams , are you running PowerShell as Administrator? The "x" is also missing in front of your "64".

I uninstalled Azure CLI and reinstalled the 64-bit version with this script:

```powershell
$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindowsx64 -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
```

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/58762114/025fd8b2-cf1a-4b43-9659-64b83e2089a5)

**Comment by alannwilliams** [comment]
That was an odd cut&paste on my part - i went back to both my powershell history and my notepad++ notes page and both had the x64 at the end. I think I just cut/pasted from the MS learning page (which omitted the x64) and I added it incorrectly by freehand into this github- not what I actually ran. 
there may be enterprise networks/vpn considerations why the 'quiet' version didn't work for me. But it might be useful to add another option to the learning page if it doesn't work for another user. Thanks for looking at this.
